db/migrations/20240523181636_userTable.sql

```diff
@@ -0,0 +1,7 @@
+-- Ensure the UUID extension is enabled
+CREATE EXTENSION IF NOT EXISTS "uuid-ossp";
+
+CREATE TABLE Users (
+  user_id UUID PRIMARY KEY DEFAULT uuid_generate_v4(),
```

Comment:
This schema is perfect!

Just wanted to call out this line, and why I prefer this style of `user_id` (as opposed to just `id`)!

I've often done an `id` here, and then when we need to refer to it in a different model used `user_id` there. And this _works_ but it makes one specific thing annoying! When you need to do a SQL join its more annoying to type the relation.

Lets assume we have a Comments table just as an example. To join Users and Comments the `id` and `user_id` way would look like this:
```
JOIN Comments on Comments.user_id = Users.id
```

And if we have `user_id` on both side we can use this `using` keyword!

```
JOIN Comments using (user_id)
```

Which is definitely not the biggest deal, but I like being able to use the `using` syntax!

Reply:
Awesome! Thank you!